### DISCUSSION TITLE
Is Github not Working with Ipynb

### DISCUSSION THREAD

**Discussion by arssite** [community/community#124597] [Other Feature Feedback, Questions, & Ideas] [answered]
### GithubDown 

Is Github have some issue with ipynb files

### Body

https://x.com/arssite_/status/1792072044143956010
Hey @github @ashtom
, it looks like there's an issue with code visibility on the platform and .ipynb files are not opening

**Comment by athrv** [comment]
Yes, it says: An error ocurred.
Please let me know if there's any solution for it

**Reply by arssite** [reply]
Yes bro

**Comment by davevad93** [comment, marked as answer]
Hi @arssite ,

Yes, that's an issue with GitHub rendering `ipynb` files, a workaround would be to use [nbviewer](https://nbviewer.org/) just paste the jupyter file link to view it. Another solution would be to use [ReviewNB](https://www.reviewnb.com/?utm_source=reviewnb_blog) which is available on GitHub [marketplace](https://github.com/marketplace/review-notebook-app/plan/MDIyOk1hcmtldHBsYWNlTGlzdGluZ1BsYW4yMDIz#plan-2023), this one also allows to view `ipynb` files from private repos while nbviewer only allows to view those from public ones.
